keyboards/handwired/asesmero/6x6/readme.md

```diff
@@ -0,0 +1,23 @@
+# Asesmero 6x6
+
+! Asesmero 6x6(https://bit.ly/3mM9xPz)
```

Comment:
This should be rehosted on Imgur or the like (if imgur, make sure it's submitted by an account)